examples/seismic/tti/tti_example.py

```diff
@@ -62,22 +56,24 @@ def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
                         help="Enable autotuning for block sizes")
     parser.add_argument("-to", "--time_order", default=2,
                         type=int, help="Time order of the simulation")
-    parser.add_argument("-so", "--space_order", default=4,
+    parser.add_argument("-so", "--space_order", default=8,
                         type=int, help="Space order of the simulation")
-    parser.add_argument("--nbpml", default=10,
+    parser.add_argument("--nbpml", default=40,
```

Comment:
why?

Reply:
ABC coefficients designed for 40, so better leave it that way, can put it back to 10.